[2024-06-05T14:13:43.804+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-05T14:13:43.829+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: xdag_id_1.gett_current_user manual__2024-06-05T14:13:35.303976+00:00 [queued]>
[2024-06-05T14:13:43.838+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: xdag_id_1.gett_current_user manual__2024-06-05T14:13:35.303976+00:00 [queued]>
[2024-06-05T14:13:43.839+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-05T14:13:43.854+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): gett_current_user> on 2024-06-05 14:13:35.303976+00:00
[2024-06-05T14:13:43.861+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=5337) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-05T14:13:43.862+0000] {standard_task_runner.py:63} INFO - Started process 5339 to run task
[2024-06-05T14:13:43.863+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'xdag_id_1', 'gett_current_user', 'manual__2024-06-05T14:13:35.303976+00:00', '--job-id', '515', '--raw', '--subdir', 'DAGS_FOLDER/jobs_dag.py', '--cfg-path', '/tmp/tmp0whymnl_']
[2024-06-05T14:13:43.866+0000] {standard_task_runner.py:91} INFO - Job 515: Subtask gett_current_user
[2024-06-05T14:13:43.919+0000] {task_command.py:426} INFO - Running <TaskInstance: xdag_id_1.gett_current_user manual__2024-06-05T14:13:35.303976+00:00 [running]> on host db8c7b709f57
[2024-06-05T14:13:44.035+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='xdag_id_1' AIRFLOW_CTX_TASK_ID='gett_current_user' AIRFLOW_CTX_EXECUTION_DATE='2024-06-05T14:13:35.303976+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-05T14:13:35.303976+00:00'
[2024-06-05T14:13:44.038+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-05T14:13:44.039+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-06-05T14:13:44.041+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'whoami']
[2024-06-05T14:13:44.059+0000] {subprocess.py:86} INFO - Output:
[2024-06-05T14:13:44.069+0000] {subprocess.py:93} INFO - default
[2024-06-05T14:13:44.070+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-06-05T14:13:44.071+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-05T14:13:44.112+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=xdag_id_1, task_id=gett_current_user, run_id=manual__2024-06-05T14:13:35.303976+00:00, execution_date=20240605T141335, start_date=20240605T141343, end_date=20240605T141344
[2024-06-05T14:13:44.168+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-05T14:13:44.209+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-05T14:13:44.212+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
